R/mod_data_management.R

```diff
@@ -114,18 +128,17 @@ mod_data_management_server <- function(id, db_conn, filter_data) {
         scrollX = TRUE
       )
     )
-    
+
     # create a proxy data to track the UI version of the table when edited
     proxy <- DT::dataTableProxy(ns("pat_table"))
-    
+
 
     # Edit a row and effect it in the UI view ####
 
     observeEvent(input$pat_table_cell_edit, {
-      
-      info <- input$pat_table_cell_edit         # get the edited row information
+      info <- input$pat_table_cell_edit # get the edited row information
 
-      info$value <- sapply(info$value, html_decoder, USE.NAMES = F) # decode any html character introduced to the values 
+      info$value <- sapply(info$value, html_decoder, USE.NAMES = F) # decode any html character introduced to the values
```

Comment:
Please use `FALSE` and not `F`

Reply:
fix here https://github.com/CDU-data-science-team/experiencesdashboard/pull/134/commits/015d9b6cbacc92928eb8de50050c43f9410664a9